workspaces/rollbar/plugins/rollbar/src/hooks/useTopActiveItems.ts

```diff
@@ -23,14 +23,15 @@ import { useApi } from '@backstage/core-plugin-api';
 
 export function useTopActiveItems(entity: Entity) {
   const api = useApi(rollbarApiRef);
-  const { organization, project } = useProjectSlugFromEntity(entity);
+  const { organization, project, environment } =
+    useProjectSlugFromEntity(entity);
   const { value, loading, error } = useAsync(() => {
     if (!project) {
       return Promise.resolve([]);
     }
 
     return api
-      .getTopActiveItems(project, 168)
+      .getTopActiveItems(project, 168, environment)
```

Comment:
Won't this be undefined as we don't set the default of production anywhere?

Reply:
Yup, not sure why this didn't show up in my queue yesterday when doing reviews but looking at it now.